encord_agents/cli/test.py

```diff
@@ -77,8 +77,8 @@ def local(
         request = requests.Request(
             "POST",
             f"http://localhost:{port}{target}",
-            data=payload,
-            headers={"Content-type": "application/x-www-form-urlencoded"},
+            json=payload,
+            headers={"Content-type": "application/json"},
```

Comment:
Have you been corresponding with @clinton-encord as to what the right Content-type is?

Reply:
Yes - See ED-319 on Linear.